src/precice/impl/ParticipantImpl.cpp

```diff
@@ -1618,4 +1643,67 @@ ParticipantImpl::MappedSamples ParticipantImpl::mappedSamples() const
   return res;
 }
 
+// Reinitialization
+
+int ParticipantImpl::getTotalMeshChanges() const
+{
+  PRECICE_TRACE();
+  PRECICE_ASSERT(_allowsRemeshing);
+  Event e("intra-handshake", profiling::Synchronize);
```

Comment:
```suggestion
  Event e("reinit-intra-handshake", profiling::Synchronize);
```

Reply:
These were placeholder names and should be changed.

What about
```suggestion
  Event e("coordinate-local-mesh-changes", profiling::Synchronize);
```

Reply:
Done